File: cakk-api/src/main/java/com/cakk/api/mapper/SearchMapper.java
package com.cakk.api.mapper;

import java.util.List;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;

import com.cakk.api.dto.response.search.TopSearchedListResponse;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class SearchMapper {

	public static TopSearchedListResponse supplyTopSearchedListResponseBy(final List<String> keywordList) {
		return new TopSearchedListResponse(keywordList, keywordList.size());
	}
}

File: cakk-core/src/main/kotlin/com/cakk/core/mapper/SearchMapper.kt
package com.cakk.core.mapper

import com.cakk.core.dto.response.search.TopSearchedListResponse

fun supplyTopSearchedListResponseBy(keywordList: List<String>): TopSearchedListResponse {
	return TopSearchedListResponse(keywordList, keywordList.size)
}
